docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,347 @@
+    account. The [VMO pack](../vm-management.md) version 4.2.0 does this automatically through its default charts
+    specification. This configuration uses the `vmEnabledNamespaces` option to specify the namespaces for which the
+    permissions are configured.
+
+    <br />
+
+    ```yaml {8}
+    charts:
+      virtual-machine-orchestrator:
+        # This namespace will be used to store golden images
+        goldenImagesNamespace: "vmo-golden-images"
+
+        # These namespaces will be created and set up to deploy VMs into
+        vmEnabledNamespaces:
+          - "default"
+          - "virtual-machines"
+    ```
+
+4.  The CDI contains an internal database of CSIs, the CSI supported access modes, volume modes and CSI-assisted cloning
+    support. However, it is still recommended to create a `StorageProfile` to force these parameters to your desired
+    values.
+
+    Add the following code snippet in another manifest file in the same cluster profile.
+
+    This snippet has three notable configurations for the storage class it defines.
+
+    - The storage class has the name `ceph-block`.
+    - The storage class supports the `Block` volume mode with the `ReadWriteMany` access mode.
+    - The storage class uses `csi-clone` for cloning operations.
+
+    <br />
+
+    ```yaml {4,8,10,11}
+    apiVersion: cdi.kubevirt.io/v1beta1
+    kind: StorageProfile
+    metadata:
+      name: ceph-block
+    spec:
+      claimPropertySets:
+        - accessModes:
+            - ReadWriteMany
+          volumeMode: Block
+      cloneStrategy: csi-clone
+    ```
+
+5.  The [Storage Profiles](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md) page
+    in the CDI documentation contains more information about storage profiles.
+
+    If Storage Profiles are not manually defined, the CDI will automatically generate one for every Storage Class, based
+    on the information in its internal database. It is recommend to verify these parameters are accurate for your use
+    case and if not, overwrite them by defining the Storage Profile manually.
+
+    As noted in the
+    [CDI documentation](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md#storage),
+    you can optionally replace the target `pvc` sections in your `vmTemplate` resources with a `storage` section once
+    you have defined a storage profile.
+
+    Using storage instead of PVC will automatically discover default values for `accessMode`, `volumeMode` and
+    optionally even the size from the source PVC and its associated storage profile.
+
+    A benefit of `storage` over `pvc` is that the CDI will automatically calculate the size overhead for `Filesystem`
+    mode storage classes, allowing the VM template to use the original disk size instead of one adjusted for the
+    filesystem overhead.
+
+    If you want to use this template, replace the contents of the second manifest with the following snippet.
```

Comment:
This steps feels a bit confusing to me. It doesn't follow as part of the existing procedure, but instead offers an alternative to the information provided in step two (I think? not sure if that's where the second manifest is). Let's discuss it and see if we can move it closer to the manifest it suggests updating?

Reply:
Yes it's an alternative to step 2 but it requires step 4.

Reply:
Perhaps I could move step 2 down instead?

Reply:
If the order of operation doesn't matter here, then maybe moving step 2 down would make more sense. This way, you can clearly identify this info as an alternative

Reply:
Added the details like we said.